tests/roles/swift_adoption/tasks/main.yaml

```diff
@@ -89,3 +89,24 @@
   register: swift_responding_result
   until: swift_responding_result is success
   retries: 15
+
+- name: Enable Glance is swift is set as backend
```

Comment:
Glance adoption comes after Swift adoption, both in docs and in test suite order. We should remove this and just adopt Glance with `replicas: 1` from the start, as Swift is already adopted at that point?

Reply:
aah ok great, I assumed glance was one of the first services being adopted, so I adapted the code for that kind of situation.
For the same reason, do we want to adopt cinder first and then glance just because cinder can be used as a glance backend? I'm fine setting it to `replicas: 0` and patch it when we know that `cinderVolume` is active. 
For this patch I agree we don't need this kind of orchestration. Updating it in my next PS.